language/benchmarks/src/bench.move

```diff
@@ -0,0 +1,73 @@
+// `cargo bench` will call a public function in this module.
+// The Module must be called `Bench` and the set of public functions are callable from the bench (Rust code).
+// `benches/transaction.rs` contains the calling code.
+// The idea is that you build your scenario with a public entry point and a bunch of private functions as needed.
+module Bench {
+
+    //
+    // Global helpers
+    //
+    assert(check: bool, code: u64) {
```

Comment:
Also, if you use `Transaction::assert` it will get inlined

Reply:
I just did not compile yet `Transaction::assert` or the standard library thus I copied it ;)

Reply:
Sounds good, just a heads up that the perf will change between the two